tekton/build-publish-images-manifests.yaml

```diff
@@ -133,6 +133,9 @@ spec:
       # This is currently the case for `cri-o` (and most likely others)
       kustomize build ${PROJECT_ROOT}/config/${KUBE_DISTRO}/overlays/default | ko resolve --platform=$(params.platforms) --preserve-import-paths -f - > $OUTPUT_RELEASE_DIR/${FILENAME_PREFIX}release.notags.yaml
 
+      # Generate separate CRD manifests
+      kustomize build ${PROJECT_ROOT}/config/crs/${KUBE_DISTRO}/ | ko resolve --platform=$(params.platforms) --preserve-import-paths -f - > $OUTPUT_RELEASE_DIR/${FILENAME_PREFIX}crds.yaml
```

Comment:
`config/crs/` has examples of CRDs in use (ie, CRs). 

i believe to achieve what we are doing here, either we will have to 

filter the output of 
`kustomize build ${PROJECT_ROOT}/config/${KUBE_DISTRO}/overlays/default` on condition `kind: CRD`

or 

restructure the `config/` tree to make a config/<platform>/crds base (or overlay) available.

Reply:
or retructure the CRDs as [kustomize components](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/components.md)

cc @vdemeester

Reply:
@jacksgt Yes, I don't see any reason not to add `yq` into `ko-gcloud` 👼🏼